x/posts/client/rest/rest.go

```diff
@@ -24,7 +24,8 @@ type CreatePostReq struct {
 	Subspace       string            `json:"subspace"`
 	OptionalData   map[string]string `json:"optional_data"`
 	CreationTime   time.Time         `json:"creation_time"`
-	Medias         types.PostMedias  `json:"medias,omitempty"`
+	Medias         *types.PostMedias `json:"medias,omitempty"`
```

Comment:
Do we need a pointer here even if it's an array as you stated previously?

Reply:
no we don't, as long as we can checks nullability over that `Medias`